### ISSUE TITLE
Proposal for new a11y check to disallow invalid ARIA attributes

### ISSUE THREAD

**Issue by judithhinlung** [ember-template-lint/ember-template-lint#2275]
Currently, Ember-template-lint does not prevent invalid ARIA attributes, or any ARIA-* properties not found in the [WAI-ARIA States and Properties spec](https://www.w3.org/WAI/PF/aria-1.1/states_and_properties), on an element. Using invalid ARIA attributes reduces usability by failing to provide the element it belongs to with the intended accessibility function.

This proposes the authoring of a new rule that enforces that ARIA attributes should be valid, and invalid attributes will generate an error. This rule is analogous to JSXs A11y [aria-props]((https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/aria-props.md) rule.

Reference:
[Supported States and Properties | Accessible Rich Internet Applications (WAI-ARIA) 1.0](https://www.w3.org/TR/wai-aria-1.0/states_and_properties)
